src/Test/Perf/Runner/Perf.Runner.csproj

```diff
@@ -12,6 +12,7 @@
     <RuntimeIdentifiers>win7;ubuntu.14.04;osx.10.10</RuntimeIdentifiers>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <SignAssembly>false</SignAssembly>
+    <Nonshipping>Perf.Runner.csproj</Nonshipping>
```

Comment:
"true"?

Reply:
Fixed.

---
In reply to: [147291094](https://github.com/dotnet/roslyn/pull/22850#discussion_r147291094) [](ancestors = 147291094)